exonum-light-client/README.md

```diff
@@ -139,4 +138,4 @@ Apache 2.0 - see [LICENSE](../LICENSE) for more information.
 [protobuf]: https://developers.google.com/protocol-buffers/docs/proto3
 [standard-serializers]: https://exonum.com/doc/api/java-binding-common/0.6.0/com/exonum/binding/common/serialization/StandardSerializers.html
 [send-tx-it]: ./src/test/java/com/exonum/client/ExonumHttpClientIntegrationTest.java
-[exonum-client]: ./src/main/java/com/exonum/client/ExonumClient.java
+[exonum-client]: https://exonum.com/doc/api/java-light-client/0.2.0/com/exonum/client/ExonumClient.html
```

Comment:
It is *not* published yet, but conveniently redirects to exonum-doc, is that correct?

Reply:
correct. It'll be published on Monday

Reply:
Good, thanks!